e2e_playwright/st_dialog_test.py

```diff
@@ -123,21 +123,21 @@ def test_dialog_reopens_properly_after_dismiss(app: Page):
     """Test that dialog reopens after dismiss."""
 
     # open and close the dialog multiple times
-    for _ in range(0, 3):
+    for _ in range(0, 10):
         open_dialog_without_images(app)
-        wait_for_app_run(app, wait_delay=250)
+        wait_for_app_run(app)
 
         main_dialog = app.get_by_test_id(modal_test_id)
 
         # sometimes the dialog does not seem to open in the test, so retry opening it by
         # clicking on it. if it does not open after the second attempt, fail the test.
         if main_dialog.count() == 0:
-            app.wait_for_timeout(100)
+            # app.wait_for_timeout(100)
```

Comment:
did you still want these commented out? or existing at all?

Reply:
ahhh yeah, I don't want them at all 😄 thank you for the catch 👍